src/user_summary/helper_functions/calculate_users_contribution_distribution.py

```diff
@@ -0,0 +1,54 @@
+import toolforge
+import logging
+import user_summary.constants as constants
+
+logger = logging.getLogger('django')
+
+
+# Function that bucketizes users based on contributions
+def calculate_users_contribution_distribution():
+    message = 'Contribution distribution calculated successfully ' \
+              'for all users. Results:'
+    success = True
+    # Definition of 8 groups - list of groups - (lowerlimit, upperlimit) pair -
+    # both inclusive
+    groups = constants.CONTRIBUTION_BUCKETS
+    try:
+        conn = toolforge.connect('enwiki')
+        with conn.cursor() as cursor:
+            cursor.execute(
+                'SELECT COUNT(*), SUM(user_editcount>0), '
+                'SUM(user_editcount >= %s AND user_editcount <= %s), '
```

Comment:
Might want to write this in a way that does not make any assumption about the number of groups.

Reply:
Okay.